lib/runAll.js

```diff
@@ -127,10 +141,14 @@ module.exports = async function runAll(
         title: 'Preparing...',
         task: () => git.stashBackup()
       },
-      {
-        title: 'Running tasks...',
-        task: () => new Listr(listrTasks, { ...listrOptions, concurrent: true, exitOnError: false })
-      },
+      ...(listrTasks.length === 1
```

Comment:
Why spreading here?

Reply:
`listrTasks` is an array of the chunked tasks, so I spread it to the main listr tasks array.

Reply:
This is for whatever reason invalid syntax:

```js
[
    listrTasks.length === 1
        ? { ...listrTasks[0], title: 'Running tasks...' }
        : ...listrTasks
]
```

So the way I see it, that spreading is the only way.

Reply:
Worked around by improving the code a bit.